app/javascript/react/components/Calendar/Calendar.style.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components";
+
+import { grey100, grey400 } from "../../assets/styles/colors";
+
+interface dayProps {
+  color?: string;
+  $hasBackground: boolean;
+  $isCurrentMonth: boolean;
+}
+
+interface labelProps {
+  $isVisible: boolean;
+  $isGrayedOut?: boolean;
+}
+
+let DAY_GAP = "6px";
```

Comment:
Please use const instead:

```suggestion
const DAY_GAP = "6px";
```
BTW :) https://www.youtube.com/watch?v=9WIJQDvt4Us cc. @jankrzempek